loki: alerting: simplified config

gabor opened the pull request:
there are two params that we handle in the loki alerting code that are not really used, i adjusted them a little:

- `interval`. alerting code currently only deals with queries that return metric data, and for those, `interval` is not used. but, truth is, even for logs, grafana does not use `interval`. we do not use it anywhere.
  - so i thought it's better to just not have it there at all as a separate variable
- `limit`. alerting code currently only deals with queries that return metric data, and for those, `limit` is not used.
  - even if it is not used, we have to set a value. the current value is `1000`, i adjusted it to `1`
    - i think it does not matter what value it has, but, if some bug happens and it starts to matter, this way we find out sooner 😁 .
      - btw. first i tried to set it to `0`, but then the loki-code complained that it has to be a positive value. i guess you can not-send-it, but if you do send it, it cannot be zero. unfortunately there does not seem to be a way to not-send-it :-(, so we go with the smallest possible number.

gabor committed:
loki: alerting: simplified config

ivanahuckova reviewed (commented):
I personally think this is less clear. I would maybe improve comments, but I think having both comments is helpful here, because they explain why are we mocking the values. You can maybe add someone else as reviewer, to hear other opinions.

gabor committed:
adjusted config handling

gabor commented:
> I personally think this is less clear. I would maybe improve comments, but I think having both comments is helpful here, because they explain why are we mocking the values. 

@ivanahuckova 
i see your point. i updated the code (moved the interval-variable "back" to be a separate variable), and added comments that i think explain what is happening better. what do you think?
